Livecheckables/libsvm.rb

```diff
@@ -0,0 +1,6 @@
+class Libsvm
+  livecheck do
+    url :homepage
+    regex(/The current release \(Version v?(\d+(?:\.\d+)+),/i)
```

Comment:
```suggestion
    regex(/The current release \(Version v?(\d+(?:\.\d+)+)[, )]/i)
```

This is just a small tweak to allow either a comma, space, or closing parenthesis as the trailing delimiter, in case the format changes slightly.